### PR TITLE
PYIC-3835: Added welsh translations for new pages

### PR DESCRIPTION
**Pull request by RogVanGDS**
### What changed

Added welsh translations for new delete user details screens

### Why did it change

To enable Welsh translations for this user flow.

### Issue tracking

- [PYIC-3835](https://govukverify.atlassian.net/browse/PYIC-3835)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or change

[PYIC-3835]: https://govukverify.atlassian.net/browse/PYIC-3835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/cy/translation.json [hunk 1]
```diff
@@ -368,50 +368,50 @@
...
+      "title": "Profi eich hunaniaeth eto gyda manylion newydd",
+      "header": "Profi eich hunaniaeth eto gyda manylion newydd",
+      "content": {
+        "paragraph1": "I ddiweddaru eich manylion, mae angen i chi: ",
+        "list1": "dileu eich manylion ",
+        "list2": "profi eich hunaniaeth eto gyda manylion newydd",
+        "paragraph2": "Byddwch angen:",
+        "list3": "ID gyda llun gyda'r enw rydych chi ei eisiau GOV.UK One Login ei ddefnyddio",
+        "list4": "dweud wrthym ble rydych chi wedi byw am y 3 mis diwethaf",
+        "paragraph3": "Darganfyddwch <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">pa ID gyda llun y gallwch ei ddefnyddio i brofi eich hunaniaeth (agor mewn tab newydd)</a>.",
```
Comment (DanCorderIPV) [inline src/locales/cy/translation.json:380]: Should all the text be inside the \<a> tag?

File: src/locales/cy/translation.json [hunk 2]
```diff
@@ -368,50 +368,50 @@
...
-        "paragraph4": "TEMP Find out <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">what photo ID you can use to prove your identity (opens in new tab)</a>.",
-        "submitButtonText": "TEMP Continue to prove your identity again"
+      "title": "Rydych chi wedi dileu'r manylion yn eich GOV.UK One Login",
+      "header": "Rydych chi wedi dileu'r manylion yn eich GOV.UK One Login",
+      "content": {
+        "paragraph1": "Rydych wedi derbyn e-bost cadarnhau.",
+        "paragraph2": "Nawr gallwch brofi eich hunaniaeth eto gan ddefnyddio'ch manylion newydd.",
+        "paragraph3": "Byddwch angen:",
+        "list1": "ID gyda llun gyda'r enw rydych chi ei eisiau GOV.UK One Login ei ddefnyddio ",
+        "list2": "dweud wrthym ble rydych chi wedi byw am y 3 mis diwetha",
```
Comment (DanCorderIPV) [inline src/locales/cy/translation.json:422]: Is this line missing an `i` at the beginning?

File: src/locales/cy/translation.json [hunk 3]
```diff
@@ -368,50 +368,50 @@
...
+        "paragraph4": "Darganfyddwch <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.gov.uk/using-your-gov-uk-one-login/proving-your-identity\">pa ID gyda llun y gallwch ei ddefnyddio i brofi eich hunaniaeth (agor mewn tab newydd)</a>.",
```
Comment (DanCorderIPV) [inline src/locales/cy/translation.json:423]: Should all the text here be inside the \<a> tag?

File: src/locales/cy/translation.json (hunk 2 above)
Comment (RogVanGDS) [reply src/locales/cy/translation.json:422]: I don't believe so. The copy on the designs is different to the copy that UCD requested in the translation file. So I chose the design copy as the correct one.

The copy that matches the design is also translated in the spreadsheet, so I chose that version.

File: src/locales/cy/translation.json (hunk 1 above)
Comment (RogVanGDS) [reply src/locales/cy/translation.json:380]: Not according to the designs in the ticket?

File: src/locales/cy/translation.json (hunk 3 above)
Comment (RogVanGDS) [reply src/locales/cy/translation.json:423]: Again, not according to the designs

File: src/locales/cy/translation.json (hunk 1 above)
Comment (DanCorderIPV) [reply src/locales/cy/translation.json:380]: Fair enough, the translation table doesn't seem to agree with the screenshots...

Comment (RogVanGDS) [reply src/locales/cy/translation.json:380]: Yeah I'm not sure who to believe in this situation, @blakeyp ?

**Comment by DanCorderIPV** [comment]
It doesn't look like these pages have their error text set (in English or Welsh), have you tested what happens if there is a validation error? Does it use some default text?

File: src/views/ipv/pyi-new-details.njk [hunk 4]
```diff
@@ -4,6 +4,11 @@
 {% set pageTitleName = 'pages.pyiNewDetails.title' | translate %}
 {% set googleTagManagerPageId = "pyiNewDetails" %}
 
+{% set errorState = pageErrorState %}
```
Comment (AmritSidhu) [inline src/views/ipv/pyi-new-details.njk:7]: You'll also need to refactor the way we set the form in the template which has changed, you can see via the other pages currently in there

**Review by DanCorderIPV — APPROVED** [review]

**Pull request merged**